### DISCUSSION TITLE
using an empty namespace in a different assembly does not produce a helpfull error message

### DISCUSSION THREAD

**Discussion by gafter** [dotnet/csharplang#8721] [Q&A]
@pdelvo commented on [Sat May 23 2015](https://github.com/dotnet/roslyn/issues/3063)

Consider this code:

``` csharp
using Assembly2;

public class Class1
{
    static unsafe void Main()
    {
    }
}
```

``` csharp
namespace Assembly2
{
}
```

Both files have to be in different projects. When you try to compile it in VS or with msbuild, csc will just return error code 1. The problem is the using Assembly2 in the first file. The namespace Assembly2 is empty.

---

@gafter commented on [Tue May 26 2015](https://github.com/dotnet/roslyn/issues/3063#issuecomment-105685983)

@MadsTorgersen What does the language spec say about the _existence_ of `Assembly2`? The compilers currently consider an imported namespace to exist only when it is nonempty.

**Comment by BillWagner** [comment]
@MadsTorgersen Should this move to the dotnet/csharpstandard repo?
